cli/popper/commands/cmd_run.py

```diff
@@ -120,7 +120,21 @@
 def cli(ctx, step, wfile, debug, dry_run, log_file, quiet, reuse,
         engine, resource_manager, skip, skip_pull, skip_clone,
         substitution, allow_loose, with_dependencies, workspace, conf):
-    """Runs a Popper workflow. Only execute STEP if given."""
+    """Runs a Popper workflow. Only execute STEP if given.
+    
+    `popper run` allows specifying the engine and the resource manager
```

Comment:
couple of comments:

  * we can use "This command" instead of `popper run`, as this is the help message for `popper run`.
  * the list mentions "two different ways" but the list has four bullet points. So we can either turn everything into a paragraph, or only show the two points and turn the last two points into a paragraph.
  * For the second bullet point, we can say through the "through a configuration file specified with the `--conf` flag"
  * For the third point, instead of "Otherwise", we can say "If none of the above are given, Popper uses..."